package-lock.json

```diff
@@ -1,8 +1,12797 @@
 {
   "name": "gtsosu",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
can you un-commit package-lock.json for now? we talked about this in the dev channel, and i wanna get our node versioning etc. sorted out and then i'll update package-lock in a separate commit